rmgpy/statmech/conformerTest.py

```diff
@@ -36,75 +36,80 @@
 
+
 class TestConformer(unittest.TestCase):
     """
     Contains unit tests of the :class:`Conformer` class.
     """
-    
+
     def setUp(self):
         """
         A function run before each unit test in this class.
         """
         self.ethylene = Conformer(
-            E0 = (0.0,"kJ/mol"),
-            modes = [
-                IdealGasTranslation(mass=(28.03,"amu")),
-                NonlinearRotor(inertia=([3.41526,16.6498,20.065],"amu*angstrom^2"), symmetry=4),
-                HarmonicOscillator(frequencies=([828.397,970.652,977.223,1052.93,1233.55,1367.56,1465.09,1672.25,3098.46,3111.7,3165.79,3193.54],"cm^-1")),
+            E0=(0.0, "kJ/mol"),
+            modes=[
+                IdealGasTranslation(mass=(28.03, "amu")),
+                NonlinearRotor(inertia=([3.41526, 16.6498, 20.065], "amu*angstrom^2"), symmetry=4),
+                HarmonicOscillator(frequencies=([828.397, 970.652, 977.223, 1052.93, 1233.55,
+                                                 1367.56, 1465.09, 1672.25, 3098.46, 3111.7, 3165.79, 3193.54], "cm^-1")),
             ],
-            spinMultiplicity = 1,
-            opticalIsomers = 1,
+            spinMultiplicity=1,
+            opticalIsomers=1,
         )
         self.oxygen = Conformer(
-            E0 = (0.0,"kJ/mol"),
-            modes = [
-                IdealGasTranslation(mass=(31.99,"amu")),
-                LinearRotor(inertia=(11.6056,"amu*angstrom^2"), symmetry=2),
-                HarmonicOscillator(frequencies=([1621.54],"cm^-1")),
+            E0=(0.0, "kJ/mol"),
+            modes=[
+                IdealGasTranslation(mass=(31.99, "amu")),
+                LinearRotor(inertia=(11.6056, "amu*angstrom^2"), symmetry=2),
+                HarmonicOscillator(frequencies=([1621.54], "cm^-1")),
             ],
-            spinMultiplicity = 3,
-            opticalIsomers = 1,
+            spinMultiplicity=3,
+            opticalIsomers=1,
         )
-        
+
         # The following data is for ethane at the CBS-QB3 level
         self.coordinates = numpy.array([
-            [  0.0000,  0.0000,  0.0000],
-            [ -0.0000, -0.0000,  1.0936],
-            [  1.0430, -0.0000, -0.3288],
-            [ -0.4484,  0.9417, -0.3288],
-            [ -0.7609, -1.2051, -0.5580],
-            [ -0.7609, -1.2051, -1.6516],
-            [ -0.3125, -2.1468, -0.2292],
-            [ -1.8039, -1.2051, -0.2293],
+            [0.0000,  0.0000,  0.0000],
+            [-0.0000, -0.0000,  1.0936],
+            [1.0430, -0.0000, -0.3288],
+            [-0.4484,  0.9417, -0.3288],
+            [-0.7609, -1.2051, -0.5580],
+            [-0.7609, -1.2051, -1.6516],
+            [-0.3125, -2.1468, -0.2292],
+            [-1.8039, -1.2051, -0.2293],
         ])
         self.number = numpy.array([6, 1, 1, 1, 6, 1, 1, 1])
-        self.mass = numpy.array([12, 1.007825, 1.007825, 1.007825, 12, 1.007825, 1.007825, 1.007825])
+        self.mass = numpy.array([12, 1.007825, 1.007825, 1.007825,
```

Comment:
If this fits nicely into one line within the recommended max width, I prefer to keep the original version (less lines)

Reply:
I am using Atom instead of PyCharm and the AutoPEP8 in Atom uses line width = 79 characters. I think it might be the reason my package suggested an extra line here. But it shouldn't be a big deal. I will keep your modification.